src/components/Badges/BadgeCheckboxes/BadgeCheckboxesForm.component.js

```diff
@@ -84,7 +84,7 @@ const BadgeCheckboxesForm = ({ checkboxValues, id, onChange, onSubmit, value, fe
 				docked={false}
 				iconAlwaysVisible
 				id={`${badgeCheckBoxesFormId}-filter`}
-				placeholder={t('FIND_COLUMN_FILTER_PLACEHOLDER', {
```

Comment:
why not setting a specific translation here instead?

Reply:
not sure I get it right

do you mean replacing 'Find entity' with hard-coded 'Find workspace' (the label that we need in our case in TMC)? this way it will be incorrect for other cases, where this checkboxes may be used in future

Reply:
Exactly but you can specify a new workspace for your list I guess
https://react.i18next.com/guides/multiple-translation-files
It was just an idea, I didn't test it yet, to be honest